crates/sui-graphql-rpc/src/config.rs

```diff
@@ -258,6 +261,21 @@ impl Default for InternalFeatureConfig {
     }
 }
 
+#[derive(Serialize, Clone, Deserialize, Debug, Eq, PartialEq)]
+pub struct TxExecFullNodeConfig {
+    #[serde(default)]
+    pub(crate) node_rpc_url: Option<String>,
+}
+
+impl Default for TxExecFullNodeConfig {
+    fn default() -> Self {
+        Self {
+            // TODO: infer default fullnode url from chain id
+            node_rpc_url: Some("https://fullnode.testnet.sui.io:443".to_string()),
```

Comment:
I think this should be `None`, there is no default RPC URL that's going to work in all cases.  The current default is going to result in some odd and behaviour if people aren't aware of it -- I would rather know that I forgot a parameter that's necessary than have it picked for me and be wrong.